spynnaker/pyNN/models/neuron/synapse_dynamics/abstract_synapse_dynamics.py

```diff
@@ -104,28 +104,28 @@ def get_provenance_data(self, pre_population_label, post_population_label):
         # pylint: disable=unused-argument
         return list()
 
-    def get_delay_maximum(self, connector, delays):
+    def get_delay_maximum(self, connector, delays, synapse_info):
```

Comment:
Similarly to the connector, this can drop the delays since it comes from synapse info